Source/Core/UICommon/GameFile.cpp

```diff
@@ -125,6 +129,8 @@ GameFile::GameFile(const std::string& path)
 
       m_volume_banner.buffer = volume->GetBanner(&m_volume_banner.width, &m_volume_banner.height);
 
+      UpdateCover();
```

Comment:
Please don't do anything over the network while creating a GameFile object, it takes too much time. You should use the same pattern as with custom banners (except skip downloading a banner if we already have downloaded one), so that the game can show up in the game list before the cover has been downloaded.